### ISSUE TITLE
Broken for layman -a?

### ISSUE THREAD

**Issue by jaccarmac** [gentoo/dotnet#367]
`layman -a` adds the old version of the repository for me, which does not include the binary versions of the .NET Core SDK and runtime. In addition, trying to add a manual overlay with an XML file gives me an error because the name of the custom overlay conflicts with the name in the Git repo. What's the best way to get this version of the overlay up and running?

**Comment by mpkh** [comment]
@jaccarmac it's updating manually, sometimes I'm late there, try now, should be synchronized (possibly I should redirect main repository in layman config)

**Comment by jaccarmac** [comment]
Thanks, ended up figuring out how to change the name on my end myself, appreciate the help!
